CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 * This file will be updated whenever a new release is put into production.
 * Any problems should be reported via the "report an issue" link in the footer of the application.
 
+## Unreleased
+### Fixed
+* Modify daily/hourly Admin buttons to use relative path via helper method ([1742](https://github.com/YaleSTC/reservations/issues/1742))
+* Changed rake daily/hourly tasks to execute immediately to fix issue with cron job in production ([#1739](https://github.com/YaleSTC/reservations/issues/1739)[#1740](https://github.com/YaleSTC/reservations/issues/1740)[#1741](https://github.com/YaleSTC/reservations/issues/1741))
+
```

Comment:
We don't need the rest of this (the Added and Removed headings) - you can just pull them out).